docs/articles/commands/multi_argument_parameters.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: articles.commands.multi_argument_parameters
+title: Multi-Argument Parameters
+---
+
+# Multi-Argument Parameters
+
+When creating a command, you may want to have a parameter that can accept multiple arguments. This is useful for commands that require a list of items, such as a list of users or a list of numbers. This was previously supported in the `CommandsNext` and `SlashCommands` extensions through the `params` keyword.
```

Comment:
(it was, actually, not supported in SlashCommands)

Reply:
I keep thinking that my `params` PR was merged